webapp/app.ironcalc.com/frontend/src/components/FileBar.tsx

```diff
@@ -19,6 +19,8 @@ export function FileBar(properties: {
 }) {
   const hiddenInputRef = useRef<HTMLInputElement>(null);
   const [toast, setToast] = useState(false);
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
```

Comment:
I think you need to remove the previous code, the "toast". So line 21 should go. Also line 4 with the lucide svg that we don't use anymore. Finally the whole toast in lines 51 to 71 (approx)